src/app/components/elements/DateJoinWrapper.jsx

```diff
@@ -3,14 +3,14 @@ import tt from 'counterpart';
 
 export default class DateJoinWrapper extends React.Component {
     render() {
-        let date = new Date(this.props.date);
-        let monthNames = ["January", "February", "March", "April", "May", "June",
+        const date = new Date(this.props.date);
+        const monthNames = ["January", "February", "March", "April", "May", "June",
```

Comment:
Please check out my new commit, now it uses FormattedDate from react-intl lib - our preferred way to localize dates